runtime/common/src/runtime_params.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 Moonbeam Foundation.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+/// Macro to generate runtime parameters.
+#[macro_export]
+macro_rules! gen_runtime_params {
```

Comment:
This macro will become very complex when we will want to add different parameters to distincts runtimes.
Can you please duplicate the code 3 times instead?
We already duplicate a lot of codes per runtime, its better for maintainability